content/kapacitor/v1.5/tick/expr.md

```diff
@@ -1,6 +1,7 @@
 ---
 title: Lambda expressions
-
+description: TICKscript uses lambda expressions  to define transformations on data points as
+well as define Boolean conditions that act as filters.
```

Comment:
Since this is multiline, you need the `>` character after the frontmatter key and you need to indent the next two lines:

```
description: >
  TICKscript uses lambda expressions  to define transformations on data points as
  well as define Boolean conditions that act as filters.
```